www/templates/diary/infinite_scroll_list.html

```diff
@@ -32,22 +32,10 @@
         </div>
 
 		<ion-list>
-		<div collection-repeat="trip in data.displayTrips" item-height="getTripHeight(trip)">
-            <div>
-
-                <!-- <div ng-if='$index==0 || trip.prevTrip.display_date != trip.display_date'>
-                    <h2 class="hr-lines" translate> {{trip.display_date}}</h2>
-                </div> Commented out as we are not using prevTrip anymore. We should come back to 
-                this and implement some sort of horizontal line between trips-day-changing later on-->
-
-                <infinite-scroll-trip-item trip="trip" config="ui_config" surveys="surveys" map-limiter="mapLimiter"></infinite-scroll-trip-item>
-                <infinite-scroll-place-item ng-if="showPlaces" trip="trip" config="ui_config" surveys="surveys"></infinite-scroll-place-item>
-
+            <div collection-repeat="entry in data.displayTimelineEntries" item-height="getCardHeight(entry)">
+                <infinite-scroll-trip-item ng-if="entry.start_ts" trip="entry" config="ui_config" surveys="surveys" map-limiter="mapLimiter"></infinite-scroll-trip-item>
+                <infinite-scroll-place-item ng-if="showPlaces && entry.enter_ts" place="entry" config="ui_config" surveys="surveys"></infinite-scroll-place-item>
```

Comment:
We can actually take this out. From an earlier draft it's the `surveys_info.surveys` field from the config, but it's no longer being used by 'trip item' nor 'place item', since they receive the whole config now

Reply:
taken out in 6e49ea91ecafa761a47df38703c4125536424119